src/test/java/core/basesyntax/FruitTransactionTest.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.fruitshop.model.FruitTransaction;
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionTest {
```

Comment:
Models usually do not require separate test class.
Constructors/getters/setters are naturally involved when other tests are performed